.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,6 +25,8 @@ Any special considerations for deployment? Consider both:
 ### If you made changes to the app code:
 
 - [ ] Unit and functional tests pass on the development VM
+- [ ] Added Python type annotations/hints in the Python code
+- [ ] **make lint** passes for the updated code.
```

Comment:
It is not 100% clear from the PR template if we use mypy used for only the securedrop/ path (i.e. the web app) or to any .py file (for example, is securedrop-admin included?)

Reply:
@emkll it is for both, how do you want me to inform that?

Reply:
Perhaps we could simply rename the title to "If you made change to the code" instead of app code. But it's a small nit, I don't feel strongly about it :).